### PR TITLE
add simple PSE generator

### PR DESCRIPTION
**Pull request by JenkeScheen**
adds a script that generates a canonical protein-ligand view `pymol` PSE file, given a protein-ligand complex PDB file.

### CODE REVIEW

**Comment by JenkeScheen** [comment]
example CLI call:
```
python ~/asapdiscovery/asapdiscovery-dataviz/asapdiscovery/dataviz/scripts/pose_to_viz_pymol.py --viz-target SARS-CoV-2-Mpro --complex Mpro_bound.pdb --out output
```

-> outputs a `.pse` with the canonical view applied, including interactions.

File: asapdiscovery-dataviz/asapdiscovery/dataviz/gif_viz.py [hunk 1]
```diff
@@ -116,8 +121,9 @@ def __init__(
         self.start = start
         self.stop = stop
         self.interval = interval
-
+        self.bool_static_view_only = True
```
Comment (hmacdope) [inline asapdiscovery-dataviz/asapdiscovery/dataviz/gif_viz.py:124]: This is set but not re-used? I would re use in function below or remove

File: asapdiscovery-dataviz/asapdiscovery/dataviz/gif_viz.py [hunk 2]
```diff
@@ -140,7 +146,7 @@ def write_traj_visualizations(self):
             output_paths.append(output_path)
         return output_paths
 
-    def write_traj_visualization(self, traj, system, path):
+    def write_traj_visualization(self, traj, system, path, bool_static_view_only):
```
Comment (hmacdope) [inline asapdiscovery-dataviz/asapdiscovery/dataviz/gif_viz.py:149]: Set a default

```suggestion
    def write_traj_visualization(self, traj, system, path, bool_static_view_only=False):
```

OR if using an attribute it shouldn't take an argument and should just use `self.static_view_only.`

Comment (hmacdope) [inline asapdiscovery-dataviz/asapdiscovery/dataviz/gif_viz.py:149]: Can we change the name of this variable to just `static_view_only`

File: asapdiscovery-dataviz/asapdiscovery/dataviz/scripts/pose_to_viz_pymol.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    )
+    logger = logger_cls.getLogger()
+    logger.info("Running pose visualization")
+
+    # check all the required files exist
+    complex = Path(args.complex)
+    if not complex.exists():
+        raise FileNotFoundError(f"Topology file {complex} does not exist")
+    out = Path(args.out)
+    logger.info(f"Topology file: {complex}: roundtripping with OpenEye.")
```
Comment (hmacdope) [inline asapdiscovery-dataviz/asapdiscovery/dataviz/scripts/pose_to_viz_pymol.py:54]: Where is this roundtrip done?

File: asapdiscovery-dataviz/asapdiscovery/dataviz/scripts/pose_to_viz_pymol.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    gif_visualiser = GIFVisualizer(
+        [None],  # we just fill these args, they're not being used.
+        [complex],
+        [out],
+        args.viz_target,
+        frames_per_ns=0,
+        smooth=5,
+        start=0,
+        logger=logger,
+        pse=False,  # can set these to True to debug viz steps.
```
Comment (hmacdope) [inline asapdiscovery-dataviz/asapdiscovery/dataviz/scripts/pose_to_viz_pymol.py:66]: Can make a CLI arg but all good for now,

**Review by hmacdope — CHANGES_REQUESTED** [review]
Few small changes, but looking good.

File: asapdiscovery-dataviz/asapdiscovery/dataviz/gif_viz.py (hunk 1 above)
Comment (JenkeScheen) [reply asapdiscovery-dataviz/asapdiscovery/dataviz/gif_viz.py]: right, I changed it to using `self` throughout.

File: asapdiscovery-dataviz/asapdiscovery/dataviz/gif_viz.py (hunk 2 above)
Comment (JenkeScheen) [reply asapdiscovery-dataviz/asapdiscovery/dataviz/gif_viz.py]: done

File: asapdiscovery-dataviz/asapdiscovery/dataviz/scripts/pose_to_viz_pymol.py (hunk 3 above)
Comment (JenkeScheen) [reply asapdiscovery-dataviz/asapdiscovery/dataviz/scripts/pose_to_viz_pymol.py]: removed this line, was outdated.

**Comment by JenkeScheen** [comment]
all addressed @hmacdope, thanks for checking.

**Review by hmacdope — APPROVED** [review]
Approve pending CI 👍

**Pull request merged**